### ISSUE TITLE
Is this project dead?

### ISSUE THREAD

**Issue by johndoe2361289638165** [awslabs/dynamodb-data-mapper-js#138]
Latest commit on 24 Nov 2018, no 1.0 release after more than one year...is this project just some experimental stuff or is it considered to become the official data-mapper for js/ts?

**Comment by cloudsolver** [comment]
I hope not!!

**Comment by tpennetta** [comment]
I agree that if this is still being under active development by _AWS Labs_ there should be a more active commit history and progression toward a full release. This project is very valuable in the wild if this can get to 1.0.

However, it is open source, so as a community perhaps we can do our best to contribute to the cause.

**Comment by johndoe2361289638165** [comment]
Is it on the roadmap that this project will be integrated in the aws-sdk?

**Comment by LucasBadico** [comment]
Dead I think not. The guys just need some help.. I see just one guy working on this.

**Comment by kennu** [comment]
This project is the best way to use DynamoDB in TypeScript apps so I really hope AWS will continue development. Transaction support is probably the biggest thing missing right now.

**Comment by michaelwittwer** [comment]
If someone is looking for some alternative DynamoDB client, we just released our first production ready version, with some up to date features like transactions and lots more.
https://github.com/shiftcode/dynamo-easy

Perhaps two projects can benefit from each other in the future.

**Comment by ajhool** [comment]
@chrisradek @srchase @jeskew a lot of us in the typescript community rely on this project and we aren't seeing issues being answered by the aws team, is it still actively maintained or should we seek other clients?

**Comment by jthomerson** [comment]
I'm curious to hear from those who have used this project thoughts on whether [@shiftcoders/dynamo-easy](https://www.npmjs.com/package/@shiftcoders/dynamo-easy) could just replace it? That project is actively maintained and appears to solve the same problems that this project attempted to.

**Comment by LucasBadico** [comment]
the true diference is that this data-mapper library is following the data-mapper library interfaces from other languages.

**Comment by gricey432** [comment]
To bring in the important part from https://github.com/aws/aws-sdk-js/issues/2637

> Yes, we have plan to move the library to V3 after it comes to offictial.

So looks like this library is dead for now and is going to become part of v3 js sdk at some point in the future

**Comment by acomagu** [comment]
I hoooope resume development. This is the best ORM for DynamoDB because it still has clean and cleverly moduled codebase, and very simple/useful interface. This still covers almost usecases of DynamoDB and works well. It was sad if this repo continue dying.

**Comment by kennu** [comment]
It is great that this library will be part of AWS SDK v3. I hope we could get instructions here when and how to start using it.
